spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -251,7 +251,10 @@ object DatabaseTablesDAO {
 
     for (j <- i + 1 to body.size + i) {
       val value = body(body.keys.toList(j - i - 1))
-      preparedStatement.setObject(j, value)
+      // We have to check if the value is a UUID because we need to set it as an UUID object in the prepared statement
+      DAOUtils.validateIsUUID(value) match
+        case Some(uuid) => preparedStatement.setObject(j, uuid)
+        case None => preparedStatement.setObject(j, value)
```

Comment:
Is there really any difference? seems like both options lead to `Object`

Reply:
When we pass a UUID as object, the query parses the UUID to postgres uuid (ex. `"aaaaa-..."::uuid`), otherwise it will send the UUID as text/string without parsing